webapp/javascript/pages/ContinuousComparisonView.tsx

```diff
@@ -95,15 +99,26 @@ function ComparisonApp() {
               dispatch(actions.setFromAndUntil({ from, until }));
             }}
             markings={{
-              left: { from: leftFrom, to: leftUntil, color: leftColor },
-              right: { from: rightFrom, to: rightUntil, color: rightColor },
+              left: {
+                from: leftFrom,
+                to: leftUntil,
+                color: leftColor,
+                overlayColor: leftColor.alpha(0.3),
```

Comment:
What does `overLayColor` correspond to? is it this 
![image](https://user-images.githubusercontent.com/23323466/181935802-76acd018-0851-4e1c-aa91-238587c1de53.png)

Reply:
@Rperry2174 you are right, it's visible semi transparent area between selection boundaries, you can set it's color by changing `overlayColor ` prop